static/js/components/dashboard/CoursePrice.js

```diff
@@ -1,21 +1,69 @@
 // @flow
 import React from 'react';
 import _ from 'lodash';
+import ReactTooltip from 'react-tooltip';
+import IconButton from 'react-mdl/lib/IconButton';
+import moment from 'moment';
 
-import type { Course } from '../../flow/programTypes';
+import type { Course, CourseRun } from '../../flow/programTypes';
 import {
+  STATUS_NOT_OFFERED,
+  STATUS_PASSED,
+  STATUS_NOT_PASSED,
   STATUS_ENROLLED_NOT_VERIFIED,
+  STATUS_VERIFIED_NOT_COMPLETED,
   STATUS_OFFERED_NOT_ENROLLED,
+  DASHBOARD_FORMAT,
 } from '../../constants';
 import { formatPrice } from '../../util/util';
 
 export default class CoursePrice extends React.Component {
   props: {
     course: Course,
+    now: moment$Moment,
   };
 
+  renderTooltip(course: Course, now: moment$Moment, firstRun: CourseRun): React$Element<*> {
+    let text = "";
+
+    switch (course.status) {
+    case STATUS_PASSED:
```

Comment:
@amir-qayyum-khan please look at this design: https://cloud.githubusercontent.com/assets/430126/18178388/1caecd22-704c-11e6-9328-1c7fab19b421.png

not every state puts text in a tooltip. look at the "Failed" state (near the bottom). there is no tooltip and the price isn't displayed